tests/integration/security/testdata/authz/v1beta1-conditions.yaml.tmpl

```diff
@@ -54,7 +54,7 @@ metadata:
 spec:
   selector:
     matchLabels:
-      app: c
+      app: {{ .cSet }}
```

Comment:
I don't really see any advantage to templating here, if `cSet` is always `c`? Unless the meaning of `cSet` here becomes "third deployment". 

Also what does `set` mean here? Seems like it's more like `cSvc`

Reply:
According to reviews on other PRs, changing the variables to src0, src1, ..., dst0, dst1,... makes it hard to review, because it is not clear whether coverage is persistent. It is suggested to keep using existing variables. Here `cSet` denotes a third deployment, which could be App C running on a pod. When we have support for multi-VM, we could pass a VM deployment to `cSet`.

Reply:
https://github.com/istio/istio/pull/32016 also reverted some variable names back for readability concern.